### PR TITLE
CGT- resident-shares - updated accessibility report for

### PR DESCRIPTION
**Pull request by Timidigital**

### CODE REVIEW

**Review by nubz — DISMISSED** [review]

File: conf/services/capital-gains-tax-calculator-resident-shares.cy.yml [hunk 1]
```diff
@@ -8,33 +8,13 @@ serviceDescription: |
...
-    date: 2023-12-31
-  - description: "Ni wnaeth y cysylltiad i symud syth ymlaen ar gyfer gwall gweithio fel y disgwyl, gan nad oedd y Dynodydd Defnyddiwr yr oedd y ‘href’ yn cyfeirio ato yn bodoli. Nid yw hyn yn bodloni maen prawf llwyddiant 2.4.1 (bypass blocks) Canllawiau Hygyrchedd Cynnwys y We, fersiwn 2.1."
-    date: 2023-12-31
-  - description: "Methodd lliw’r dangosydd ffocws a ddefnyddiwyd ar draws y gwasanaeth â bodloni’r gymhareb ddisgwyliedig o 3:1 i basio maen prawf llwyddiant 1.4.11 ‘Non-text Contrast AA’, Canllawiau Hygyrchedd Cynnwys y We, fersiwn 2.1, a gall fod yn anodd i rai defnyddwyr ei ganfod. Nid yw hyn yn bodloni maen prawf llwyddiant 1.4.11 (non-text contrast) Canllawiau Hygyrchedd Cynnwys y We, fersiwn 2.1."
-    date: 2023-12-31
-  - description: "Ni ddefnyddiwyd y pennawd i gyflwyno’r cynnwys, nodwyd swm gan <h1>. Nid yw hyn yn bodloni maen prawf llwyddiant 2.4.6. (headings and labels) Canllawiau Hygyrchedd Cynnwys y We, fersiwn 2.1."
-    date: 2023-12-31
-serviceLastTestedDate: 2020-10-22
+complianceStatus: full
+serviceLastTestedDate:  2024-02-29
```
Comment (adamliptrot-oc) [inline conf/services/capital-gains-tax-calculator-resident-shares.cy.yml:12]: Can we update this to 1 March 2024

File: conf/services/capital-gains-tax-calculator-resident-shares.yml [hunk 2]
```diff
@@ -7,33 +7,13 @@ serviceDescription: |
...
-    date: 2023-12-31
-  - description: "The error skip link did not function as expected, as the ID that the ‘href’ referred to did not exist. This doesn’t meet WCAG 2.1 success criteria 2.4.1 (bypass blocks)."
-    date: 2023-12-31
-  - description: "The colour of the focus indicator used throughout the service failed to meet the expected ratio of 3:1 to pass WCAG 2.1 success criteria 1.4.11 Non-text Contrast AA and may prove difficult for some users to perceive. This doesn’t meet WCAG 2.1 success criteria 1.4.11 (non-text contrast)."
-    date: 2023-12-31
-  - description: "The heading had not been used to introduce the content, the <h1> stated an amount. This doesn’t meet WCAG 2.1 success criteria 2.4.6 (headings and labels)."
-    date: 2023-12-31
-serviceLastTestedDate: 2020-10-22
+complianceStatus: full
+serviceLastTestedDate:  2024-02-29
```
Comment (adamliptrot-oc) [inline conf/services/capital-gains-tax-calculator-resident-shares.yml:11]: Can we update this to 1 March 2024

File: conf/services/capital-gains-tax-calculator-resident-shares.cy.yml (hunk 1 above)
Comment (adamliptrot-oc) [inline conf/services/capital-gains-tax-calculator-resident-shares.cy.yml:12]: Can we update this to 1 March 2024

File: conf/services/capital-gains-tax-calculator-resident-shares.yml (hunk 2 above)
Comment (adamliptrot-oc) [inline conf/services/capital-gains-tax-calculator-resident-shares.yml:11]: Can we update this to 1 March 2024

**Review by adamliptrot-oc — DISMISSED** [review]

**Review by nubz — APPROVED** [review]

**Review by nubz — APPROVED** [review]

**Pull request merged**